fre/frepp/configUpdates/schema-pp.json

```diff
@@ -0,0 +1,137 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Schema for PP Yaml",
+  "type": "object",
+  "properties": {
+    "configuration_paths": { 
+      "type": "object",
+      "properties": {
+        "rose-suite": {"type": "string"},
+        "rose-remap": {"type": "string"},
+        "rose-regrid": {"type": "string"}
+      }
+    },
+    "analysisLen": {"type": "string"},
+    "chunk96": {"type": "string"},
```

Comment:
The name `chunk96` is part of the schema, here?

Reply:
Oo, I'll check this.

Reply:
Yeah, the `chunk96` was part of those user variables that could be reused later. This name could change

Reply:
User variable names shouldn't be in the pp.yaml schema, should they? Can users name their variables whatever they like?

Reply:
Well in the pp.yaml, in order for it to be validated, should be defined as something. I can change it to something like `pp_amip_chunk` or something instead of `chunk96`. The actual variable that can be reused throughout the pp.yaml, i.e. ` &PP_AMIP_CHUNK96`, can be called anything the user wants though, as long as it's consistent through the yaml.

Reply:
Sorry, I'm still confused. is the variable called `pp_amip_chunk` or `PP_AMIP_CHUNK96`?

Reply:
You're good! That wasn't a great explanation on my part at all. I was looking for this link to help. 

https://docs.geoserver.org/2.21.x/en/user/styling/ysld/reference/variables.html

With reusable yaml variables, it has this structure:
`define: &variable <value>`
Where: `define = pp_amip_chunk`
              `$variable = &PP_AMIP_CHUNK96` (which was what the variable was called in the XML, and the variable to be reused in the yaml later on in the components section)
              `<value> = "1yr"`

Reply:
aha, thanks for that. I see. I agree that's just what we want.

Can't we have just one variable name, `PP_AMIP_CHUNK96` that is user set? What does the lower case `pp_amip_chunk` in?

This is the stock example:
```
define: &variable <value>
```

Here's our example:
```
define: %PP_AMIP_CHUNK96 '1yr'
```

And then PP_AMIP_CHUNK96 can be used in the yaml below as:

```
*PP_AMIP_CHUNK96
```